app/routes/docs.samples.mdx

```diff
@@ -0,0 +1,176 @@
+---
+handle:
+  breadcrumb: Code Samples
+---
+
+import { Highlight } from '~/components/Highlight'
+
+# Code Samples
+
+We have collected a short list of functions and examples that may be useful
+when working with GCN Notices in your code. These include conversions from
+text/VOEvent alert formats to json and xml {'('}for VOEvent alerts{')'} and
+examples for how to save alerts in these formats.
+
+For more examples, or to contribute your own ideas, see our [Utility Sample](https://github.com/nasa-gcn/utility-samples) and [gcn-kafka-python](https://github.com/nasa-gcn/gcn-kafka-python) repositories.
```

Comment:
Change this to a link to this route module in this repo.

Reply:
Sure, which module are you referring to?

Reply:
_This_ module: app/routes/docs/samples.mdx.